SiG/laser-util-sig.py

```diff
@@ -194,6 +199,15 @@ def get_power_attenuator(self):
         print(f"laser power attenuator is 0x{value:02x}")
         return value
 
+    def get_adc_raw(self, n):
+        if n is not None:
+            self.set_selected_adc(n)
+            value = self.get_cmd(0xd5, lsb_len=2) & 0xfff
+            print("==================================")
+            print(f"Raw hex temp = {value}")
```

Comment:
This won't actually come out in hex.  Can I recommend:
`print(f"Raw hex temp = 0x{value:03x}")`